pkg/constants/constants.go

```diff
@@ -19,7 +19,9 @@ const (
 	ElasticsearchPort = "9200"
 	FluentdName       = "fluentd"
 	KibanaName        = "kibana"
-	LogStoreService   = ElasticsearchFQDN + ":" + ElasticsearchPort
+	LogStoreURL       = "tls://" + ElasticsearchFQDN + ":" + ElasticsearchPort
```

Comment:
My ignorance to protocol but is this equivalent to accessing ES using http?

Reply:
out of curiosity, do we need to specify `tls` doesn't fluentd already build its connector to ES using tls?

Reply:
If ES is using HTTP then we should definitely use http: and https: - anyonne know for sure?

I think we do need to allow TLS and non-TLS connections, unless ES requires TLS only. Dev environments and trusted clusters will want to be able to avoid the burden of managing certs.

Reply:
If that's the case we should definitely use https: instead of tcp/tls for clarity.